### ISSUE TITLE
App Information does not scroll, version info truncated

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-app-android#3369]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [ ] Bug is not already reported in another issue
This issue is closely related to https://github.com/corona-warn-app/cwa-app-android/issues/3059 "App Information cut off at bottom on small screen (2)" with Internal Tracking ID: [EXPOSUREAPP-7040](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7040).

This issue is posted separately here due to different environments.

The original issue was https://github.com/corona-warn-app/cwa-app-android/issues/2302.

## Technical details

- Device name: Google Pixel emulator<br>5.0" 1080x1920 420dpi
- Android version: 10 (API 29) or Android 11 (API 30)
- App version: 2.3.2-RC2 & 2.4.0-RC7

## Describe the bug

The App Information menu fails to scroll down in some configurations, cutting off the bottom end of the display. 

Version | Result | Status
--- | --- | ---
2.2.1 | Version displayed | :heavy_check_mark:
2.3.2-RC2 deviceForTestersDebug | Version truncated | :x:
2.4.0-RC7 deviceForTestersDebug | Version truncated | :x:

The issue is sensitive to display size and font size.

The last line of the screenshot following should show `Version: 2.4.0-RC7`.

![2 4 0-scroll-issue](https://user-images.githubusercontent.com/66998419/121063015-171a8c80-c7c6-11eb-85cd-9266252aa82b.jpg)

### Setup
1. (Android) Settings > Display > Advanced
2. Set Font size to Large and Display size to Default

### In CWA
1. Start Screen, tap three-dot symbol
2. Tap "App Information"
3. Try to scroll down to bottom of view

## Expected behavior

Using an API 29 emulator the App Information view should display the complete view including the last line which is the Version of the app. If the view is too large then it should be possible to scroll down until the app version number is visible. 

It should be possible to scroll by tapping, holding and dragging anywhere in the scrolled view, however in this exact scenario it is only possible to scroll by using the area around the version number.

## Investigation

![App Information Layout Inspector](https://user-images.githubusercontent.com/66998419/121063043-20a3f480-c7c6-11eb-97d1-4d67e03174f0.jpg)

PR https://github.com/corona-warn-app/cwa-app-android/pull/3098 from @SamuraiKek attempted to fix this issue. This did improve the situation for some configurations, but perhaps there are multiple root causes for this issue?

I am seeing a very strange effect. If I set a breakpoint on https://github.com/corona-warn-app/cwa-app-android/blob/9ec222a431340917834e831cc60edd14fa062d27/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/information/InformationFragmentViewModel.kt#L26

whilst the app is paused due to the breakpoint, the App Information can be scrolled. After resuming the execution of the app, it is only possible to scroll the App Information by grabbing the part of the view where the truncated version information is displayed.

## Possible fix

Adding the following seems to mitigate the issue, as a workaround.

```
                <Space
                    android:layout_width="match_parent"
                    android:layout_height="56dp" />
```
This is added immediately before
https://github.com/corona-warn-app/cwa-app-android/blob/af6edf5885afeae9fe0243a8f0c9b6c30e3ec5d9/Corona-Warn-App/src/main/res/layout/fragment_information.xml#L125

----
Internal Tracking-ID: [EXPOSUREAPP-7765](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7765)

**Comment by MikeMcC399** [comment]
It seems that the view **will** scroll, but only if I tap and hold in the very small space **underneath** the truncated version information and pull upwards.

**Comment by MikeMcC399** [comment]
I have added a suggestion for a workaround into the main issue submission.

**Comment by MikeMcC399** [comment]
@dsarkar
Could I get a new internal tracking ID assigned and maybe some dev feedback on this issue?

**Comment by heinezen** [comment]
@MikeMcC399 

Sorry for the late reply. We were quite busy with the releases.

I've notified the developers via Jira. I don't think there would be objections to a PR from their side, so if you still want to do that, you could go for it. Otherwise we'll notify you here about any upcoming fixes.

----
Corona-Warn-App Open Source Team

**Comment by MikeMcC399** [comment]
@heinezen 
 
> Sorry for the late reply. We were quite busy with the releases.

No problem, and I completely understand.

> I've notified the developers via Jira. I don't think there would be objections to a PR from their side, so if you still want to do that, you could go for it. Otherwise we'll notify you here about any upcoming fixes.

Thank you. I will submit a PR in the next couple of days for consideration.
> Corona-Warn-App Open Source Team

**Comment by MikeMcC399** [comment]
I did a few more tests and I have now submitted PR #3416.

**Comment by MikeMcC399** [comment]
PR #3416 has been added to [2.4.0 milestone](https://github.com/corona-warn-app/cwa-app-android/milestone/37).

**Comment by MikeMcC399** [comment]
Confirmed fixed with CWA Android 2.4.2 on

Device name: Google Pixel emulator
5.0" 1080x1920 420dpi
Android version: 10 (API 29) 

with Font size set to Large and Display size set to Default.

With CWA Android 2.3.4 on this emulator the App Information can only be scrolled by grabbing the truncated version number.
In CWA Android 2.4.2 scrolling of App Information works by tapping and holding anywhere in the App Information layout.

Note that the API 29 emulator has no Bluetooth, so there is no ENF version display in this environment.

Also checked with API 30, however scrolling worked already with 2.3.4. It continued to work after updating to 2.4.2.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for detailed feedback. I will forward your observations to the internal ticket.
